### PR TITLE
Improve dino.profile.

### PR DESCRIPTION
**Pull request by crocket**
dino.profile: Allow functionalities that require dbus

### CODE REVIEW

File: etc/profile-m-z/transmission-qt.profile [hunk 1]
```diff
@@ -11,8 +11,8 @@ private-bin transmission-qt
 
 # private-lib - breaks on Arch
 ignore private-lib
-
 ignore memory-deny-write-execute
+protocol netlink
```
Comment (rusty-snake) [inline etc/profile-m-z/transmission-qt.profile:15]:
```suggestion

protocol netlink
```

What does the absence of `netlink` break?

Comment (crocket) [reply etc/profile-m-z/transmission-qt.profile:15]: Could not create AF_NETLINK socket (Operation not supported)

Comment (rusty-snake) [reply etc/profile-m-z/transmission-qt.profile:15]: But nothing is broken, right? That's Qt. Why should we add unnecessary attack-surface?

Comment (crocket) [reply etc/profile-m-z/transmission-qt.profile:15]: I don't know whether Qt prints that. Nothing seems broken yet. But, I didn't like error messages.

Comment (kmk3) [reply etc/profile-m-z/transmission-qt.profile:15]: @crocket [on Feb 15](https://github.com/netblue30/firejail/pull/4948#discussion_r806919223):

> I don't know whether Qt prints that. Nothing seems broken yet. But, I didn't
> like error messages.
> 
> Why does Qt try to create a netlink socket?

Related thread:

* <https://github.com/netblue30/firejail/issues/4925#issuecomment-1039199543>

Comment (crocket) [reply etc/profile-m-z/transmission-qt.profile]: I deleted modification to transmission-qt.profile.

**Review by rusty-snake — APPROVED** [review]

**Review by kmk3 — APPROVED** [review]

**Comment by Fred-Barclay** [comment]
Hi @crocket - thanks for the PR! Just for documentation, can you describe what functionalities require dbus?

**Comment by crocket** [comment]
I just added comments to the code.

**Comment by Fred-Barclay** [comment]
Merged, thanks!

**Pull request merged**